dev/tidb-in-kubernetes/maintain/lightning.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Restore data into TiDB in Kubernetes
+summary: Learn how to quickly restore data into a TiDB cluster in Kubernetes with TiDB Lightning.
+category: how-to
+---
+
+# Restore data into TiDB in Kubernetes
+
+This document describes how to restore data into a TiDB cluster in Kubernetes using [TiDB Lightning](https://github.com/pingcap/tidb-lightning).
+
+TiDB Lightning contains two components: tidb-lightning and tikv-importer. In Kubernetes, the tikv-importer is inside the Helm chart of the TiDB cluster, it is deployed as a `StatefulSet` with `replicas=1` while tidb-lightning is in a separate Helm chart and deployed as a `Job`.
+
+Therefore, both the tikv-importer and tidb-lightning need to be deployed to restore data with TiDB Lightning.
```

Comment:
`tikv-importer must be enabled in tidb-cluster first` will be more correct

Reply:
emm... in below instructions, there are details on how to deploy tikv-importer, it makes sense to say `deploy tikv-importer` here.